traits/editor_factories.py

```diff
@@ -0,0 +1,207 @@
+#  Copyright (c) 2005-19, Enthought, Inc.
```

Comment:
2005-20?

Reply:
Possibly not worth fixing in this PR. I'll open a general issue for updating copyright headers and add it to the 6.0.0 release milestone.

Reply:
#692